x-pack/plugins/security/server/routes/views/logged_out.js

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export function initLoggedOutView(server) {
+  const loggedOut = server.getHiddenUiAppById('logged_out');
+
+  server.route({
+    method: 'GET',
+    path: '/logged_out',
+    handler(request, reply) {
+      return reply.renderAppWithDefaultConfig(loggedOut);
```

Comment:
Is there anything we can do with this route to prevent users from having a false sense of security? If you hit this route directly, you'll be presented with the "Successfully logged out" message, but you're actually still authenticated.

I know you can only hit this organically through the logout sequence, but if browsers end up caching the url or something, then an auto-complete power user might think this does something useful.

Reply:
We could redirect them to the app itself if they're still logged in